vllm/v1/core/kv_cache_utils.py

```diff
@@ -164,14 +164,13 @@ def get_all_free_blocks(self) -> List[KVCacheBlock]:
         return ret
 
 
-def generate_block_hash_extra_keys(
+def generate_block_hash_extra_keys_for_mm(
```

Comment:
I tried to make the name shorter and non-public to illustrate that this is not supposed to be called directly other than by `generate_block_hash_extra_keys`.
```suggestion
def _gen_mm_extra_hash_keys(
```